packages/codeeditor/src/editor.ts

```diff
@@ -451,10 +451,18 @@ namespace CodeEditor {
 
     /**
      * Repaint the editor.
+     * 
+     * #### Notes
+     * A repainted editor should fit to its host node.
      */
     refresh(): void;
 
     /**
+     * Resize the editor to fit its host node.
+     */
+    resizeToFit(): void;
```

Comment:
In the case of CodeMirror, yes, we can choose to not perform a full `refresh` until it gains focus again.